### PR TITLE
fix: Duplicate User FCMs

### PR DESCRIPTION
**Pull request by faztp12**
This will strip away FCM tokens from old users when the same token gets registered against a new user

### CODE REVIEW

File: renovation_core/utils/fcm.py [hunk 1]
```diff
@@ -90,11 +90,21 @@ def get_client_tokens(user=None):
 
 
 def _add_user_token(user, token, linked_sid=None):
-  _existing = frappe.db.get_value(
-      "FCM User Token", {"token": token, "user": user})
-  if _existing:
-    frappe.db.set_value("FCM User Token", _existing, "last_updated", now())
-    return frappe.get_doc("FCM User Token", _existing)
+  _existing = frappe.get_all(
```
Comment (assemmarwan) [inline renovation_core/utils/fcm.py:93]: @faztp12 Why using `get_all`? Isn't it guaranteed that a token will exist only once?

Comment (faztp12) [reply renovation_core/utils/fcm.py:93]: @assemmarwan yeah frappe.db.get_value can work here,
but im afraid there are tokens linked to multiple users already ?

Comment (assemmarwan) [reply renovation_core/utils/fcm.py:93]: I see. Please add some documentation about it.

Comment (faztp12) [reply renovation_core/utils/fcm.py:93]: @assemmarwan  Hm, come to think of it,
I think a patch would work better here ?

Comment (assemmarwan) [reply renovation_core/utils/fcm.py:93]: Yeah, that makes sense 👍

**Review by assemmarwan — APPROVED** [review]

**Pull request merged**